docs/framework/install/on-windows-vista.md

```diff
@@ -15,9 +15,9 @@ These instructions will help you install the .NET Framework versions you need. T
 
 ## .NET Framework 4.6
 
-The [.NET Framework 4.6](https://www.microsoft.com/download/details.aspx?id=48130&e6b34bbe-475b-1abd-2c51-b5034bcdd6d2=True) is the latest supported .NET Framework version on Windows Vista and Windows Server 2008.
+The [.NET Framework 4.6](https://dotnet.microsoft.com/download/dotnet-framework/net46&e6b34bbe-475b-1abd-2c51-b5034bcdd6d2=True) is the latest supported .NET Framework version on Windows Vista and Windows Server 2008.
```

Comment:
can't these be simplified?

```suggestion
The [.NET Framework 4.6](https://dotnet.microsoft.com/download/dotnet-framework/net46) is the latest supported .NET Framework version on Windows Vista and Windows Server 2008.
```

Reply:
@mairaw I think something happened when translating the link. I didn't account for that guid.